virtinst/devices/video.py

```diff
@@ -11,15 +11,25 @@
 class DeviceVideo(Device):
     XML_NAME = "video"
 
-    _XML_PROP_ORDER = ["model", "vram", "heads", "vgamem"]
-    model = XMLProperty("./model/@type")
+    _XML_PROP_ORDER = ["_model", "vram", "heads", "vgamem"]
     vram = XMLProperty("./model/@vram", is_int=True)
     vram64 = XMLProperty("./model/@vram64", is_int=True)
     ram = XMLProperty("./model/@ram", is_int=True)
     heads = XMLProperty("./model/@heads", is_int=True)
     vgamem = XMLProperty("./model/@vgamem", is_int=True)
     accel3d = XMLProperty("./model/acceleration/@accel3d", is_yesno=True)
 
+    def _set_model(self, val):
+        self._model = val
+        if self._model != "qxl":
+            self.ram = None
```

Comment:
would be nice if this was demonstrated in the test suite too. something to fold into the spice -> vnc testing